### PR TITLE
dioalma.protocol Update assetlist.json

### PR DESCRIPTION
**Pull request by DioAlma**
With the intention of getting veryfied status on TFM.com

### CODE REVIEW

File: terra2/assetlist.json [hunk 1]
```diff
@@ -58,6 +58,114 @@
...
+          "denom": "Dinheiros",
+          "exponent": 0
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "terra1spkm49wd9dqkranhrks4cupecl3rtgeqqljq3qrvrrts2ev2gw6sy5vz3k",
+      "base": "cw20:terra1spkm49wd9dqkranhrks4cupecl3rtgeqqljq3qrvrrts2ev2gw6sy5vz3k",
+      "name": "dinheiro",
+      "display": "Dinheiros",
+      "symbol": "Dinheiros",
```
Comment (JeremyParish69) [inline terra2/assetlist.json:78]: Please use ALL CAPS symbols

File: terra2/assetlist.json [hunk 2]
```diff
@@ -58,6 +58,114 @@
...
+      "logo_URIs": {
+        "png": "https://static.wixstatic.com/media/1e62a0_04207897171e4f5c8931295172b3e34c~mv2.png"
```
Comment (JeremyParish69) [inline terra2/assetlist.json:80]: Please upload the images for addition here (into the /images/ folder) and point these URLs to the corresponding Chain Registry location. e.g., 'https://github.com/cosmos/chain-registry/terra2/images/dinheiros.png'

File: terra2/assetlist.json (hunk 1 above)
Comment (DioAlma) [reply terra2/assetlist.json]: done.

File: terra2/assetlist.json (hunk 2 above)
Comment (DioAlma) [reply terra2/assetlist.json]: done. thank you.

File: terra2/assetlist.json [hunk 3]
```diff
@@ -58,6 +58,114 @@
...
+      "base": "cw20:terra1qj5hs3e86qn4vm9dvtgtlkdp550r0rayk9wpay44mfw3gn3tr8nq5jw3dg",
+      "name": "escudo",
+      "display": "Escudos",
+      "symbol": "ESCUDOS",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/Escudos.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/Escudos.png.png"
```
Comment (JeremyParish69) [inline terra2/assetlist.json:138]: double .png.png here

Comment (DioAlma) [reply terra2/assetlist.json]: Fixed, thank you.

**Review by DioAlma — COMMENTED** [review]
Ready for merge.

**Comment by DioAlma** [comment]
Read for merge thank you

**Pull request merged**